.github/CODEOWNERS

```diff
@@ -3,14 +3,15 @@
 # project own the whole product.
 
 # Normally, *.ui, *.cpp and *.h files are reviewed by the core-cpp team only
```

Comment:
I think this line needs revising as well...

Reply:
I think that's right, because UI team wouldn't be reviewing any headers?

Reply:
Well that `.ui` file is the Qt Designer file/form/dialogue file which is viewable in the separate Qt Designer application outside of the Qt Creator which uses the integrated plugin to provide the same viewing/editing features.  As such it would enable reviewers to see most aspects of the design of the forms/dialogues which could reasonably be part of what they are reviewing...

Reply:
I'm not sure what are you getting at, what is the issue exactly?

Reply:
If we have people *reviewing* the UI they may want to look at it! Examining the `.ui` forms with the Qt Designer utility may be one means that they can inspect part of the UI - although for things like the editor it will only show them fragments of the complete arrangement and not give them the overall effect when in use - but does make it obvious (I hope) which `class` is responsible for which bit of the UI if they are not happy about some aspect of it.

Reply:
I have no idea how that is related to Github auto-tagging which team should be notified for review :disappointed:

Reply:
Well, it depends on who is in that team and what tools/competencies they are going to have.  If they are to review the UI they may want to inspect it and then those files *might* be useful - or might not if they are not happy to go looking into those parts of the design.

If they are only considering the textual content of the user interface then it is less likely to be a concern, but it could be, I'm just not sure yet *who*/*what type of person* is going to be in this new team...

Reply:
I've created the team today: https://github.com/orgs/Mudlet/teams/ui-review/members

I don't think we have to tell people what tools they should be using in a file intended for use only by the Github infrastructure. Can I merge this?

Reply:
Yeah, I guess so.

Reply:
OK, thx!